### ISSUE TITLE
RegEx for PDB-CCD is wrong

### ISSUE THREAD

**Issue by bmeldal-eg** [identifiers-org/identifiers-org.github.io#198]
Currently: ^\w{3}$

Expected: ^\w{1,3}$

because IDs can be anything UP TO 3 letters.

Thanks.

**Comment by cthoyt** [comment]
Do you have an example identifier that is less than 3?

**Comment by bmeldal-eg** [comment]
Sorry, forgot to add:
https://www.ebi.ac.uk/pdbe-srv/pdbechem/chemicalCompound/show/A
or
https://www.ebi.ac.uk/pdbe-srv/pdbechem/chemicalCompound/show/PI

**Comment by bmeldal-eg** [comment]
Hi Charlie,
Thank you. I have a list of missing resources that we would like to have added. Do you prefer one ticket per resource or lump them all together?
Birgit

**Comment by bmeldal-eg** [comment]
Excellent. If you have a template Excel sheet that goes with the prefix request form, please send it to me: birgit.meldal AT eaglegenomics.com and I will populate it.

**Comment by cthoyt** [comment]
I don't have a template right now but will develop one (issue at https://github.com/biopragmatics/bioregistry/issues/371). In the mean time, you'll need the following columns:

1. Requested prefix
2. Name of resource
3. Homepage of resource
4. Long-form description of resource
5. Example local unique identifier from resource. More than one welcome!
6. Regular expression pattern for local unique identifier
7. URI format string (for resolving local unique identifiers to web pages)
8. Contributor name
9. Contributor ORCID id
10. Contributor GitHub

Optional fields:
1. Links to publications
2. GitHub repository for resource
3. License for resource
4. Wikidata entry for corresponding database
5. Download link for OWL/OBO/OBO Graph JSON
6. Prefix synonyms
7. Additional free text comments
8. Twitter handle for database/resource
9. FAIRSharing ID
10. BioPortal Ontology ID

**Comment by bmeldal-eg** [comment]
The new prefix is showing in [bioregistry](https://bioregistry.io/registry/pdb-ccd) but not in [identifier.org](https://registry.identifiers.org/registry/pdb-ccd)

Reading the [instructions](https://github.com/biopragmatics/bioregistry/blob/main/docs/CONTRIBUTING.md) I notice it's still "ec-code" in [identifiers.org](https://registry.identifiers.org/registry/ec-code) as well. Why do the prefixes not get propagated?

**Comment by bmeldal-eg** [comment]
But this is the identifiers.org GitHub repo...

Although, this is strange, there is this [request page](https://registry.identifiers.org/prefixregistrationrequest) as well as a link to this GH repo I'm in right now. Hmmm.

I'll try the request page, too.

**Comment by cthoyt** [comment]
Indeed. Since the identifiers.org team has been unresponsive the last year or so, I’ve been notifying people on this issue tracker that there’s a better alternative where their issues can be more promptly fixed/addressed

**Comment by hHermjakob** [comment]
Done in identifiers.org.

**Comment by bmeldal-eg** [comment]
Hi Henning :) 

Thank you for changing the RegEx. 

I'm curious though, when I entered the prefix in the form it rejected the existing "pdb-ccd" and only accepted "pdb.ccd" as per bioregistry instructions.
